src/ncorr_commandLine.cpp

```diff
@@ -553,92 +424,105 @@ int main(int argc, char *argv[]) {
 	else{
 		config_DIC_analysis = DIC_analysis_config::REMOVE_BAD_POINTS;
 	}
+	std::vector<Image2D> imgs;
+	for (json::iterator it = inputFile.find("images")->begin(); it != inputFile.find("images")->end(); ++it) {
+  		image_names.push_back(it->get<string>());
+	}
+	//string directoryInUse = getDirPath(getImagesArray[1]);

```

Comment:
Delete

Reply:
What do you want me to delete here?

Reply:
The random commented out lines of code. Unless they are meaningful?